schemas/get-client-response.yml

```diff
@@ -14,6 +14,12 @@ properties:
       Date and time where the clients access is set to expire
     type:                   string
     format:                 date-time
+  deleteOnExpiration:
```

Comment:
Property should be listed as required in the `required` property at bottom of object description...